tutorials/plot_artifacts_correction_ica.py

```diff
@@ -68,6 +71,26 @@
 
 
 ###############################################################################
+# Component properties
+# --------------------
+#
+# Let's take a closer look at three potential candidates for artifact-related
+# components: IC 12, IC 15 and IC 21
+
+# first, component 28:
```

Comment:
Sorry, I had a different number of components set in my notebook, and it seems I didn't correct IC id's everywhere when I set the number of components to be the same as in tutorial.